plugins/spu2-x/src/Windows/SndOut_XAudio2.inl

```diff
@@ -132,14 +132,13 @@ private:
             pSourceVoice->FlushSourceBuffers();
             pSourceVoice->Start(0, 0);
 
-            qbuffer = new s16[m_nBuffers * m_BufferSize];
-            ZeroMemory(qbuffer, m_BufferSizeBytes * m_nBuffers);
+            m_buffer = std::make_unique<s16[]>(m_nBuffers * m_BufferSize);
```

Comment:
```
Yes. make_unique value initialises each element to 0.
```
Is it a windows behavior ? On linux, it is only a wrapper to `unique_ptr(new...)`. I don't see any think related on it on the GCC header. Note I didn't look at the standard.